### PR TITLE
Attach hyperlink to each issue

### LINKED ISSUE #24: #8: Attach hyperlink to each issue
1) Read guideline [here](https://gist.github.com/iamnovichek/801ac395a06a7d8b04b022084ace684d)
2) Create `get_hyperlink` function in `utils.py` that returns a hyperlink string to the current issue
3) Attach the link to each issue in `send_available_issues` function [here](https://github.com/KostCode-Organization/hackathon_bot/blob/main/tracker/telegram/bot.py#L132)

### PR DESCRIPTION
**Pull request by codebestia**
Fixes #24

### CODE REVIEW

**Comment by codebestia** [comment]
@iamnovichek  Please review

File: tracker/utils.py [hunk 1]
```diff
@@ -260,3 +260,15 @@ def get_user_revisions(telegram_id: str) -> list[dict]:
                 return_data["reviews"] = reviews_data
                 reviews_list.append(return_data.copy())
     return reviews_list
+
+
+def modify_issue_title(issue_title: str, issue_link: str) -> str:
```
Comment (iamnovichek) [inline tracker/utils.py]: Please, rename it to `attach_link_to_issue`

**Review by iamnovichek — CHANGES_REQUESTED** [review]
Please, provide some screenshots to prove your changes work as expected

**Comment by codebestia** [comment]
## The screenshots

Screenshot from 2024-11-25 06-44-42
Screenshot from 2024-11-25 06-44-53

**Review by iamnovichek — APPROVED** [review]

**Comment by iamnovichek** [comment]
@codebestia approved

**Pull request merged**